hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/regression/factories/EvmAddressFuzzingFactory.java

```diff
@@ -0,0 +1,94 @@
+
+import com.hedera.services.bdd.spec.HapiSpec;
+import com.hedera.services.bdd.spec.HapiSpecOperation;
+import com.hedera.services.bdd.spec.infrastructure.OpProvider;
+import com.hedera.services.bdd.spec.infrastructure.providers.names.RegistrySourcedNameProvider;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.BiasedDelegatingProvider;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.crypto.TransferToRandomLazyCreate;
+import com.hedera.services.bdd.spec.infrastructure.selectors.RandomSelector;
+import com.hedera.services.bdd.spec.keys.SigControl;
+import com.hederahashgraph.api.proto.java.Key;
+import java.util.function.Function;
+import java.util.stream.IntStream;
+
+public class EvmAddressFuzzingFactory {
+    private static final int NUM_DISTINCT_ECDSA_KEYS = 42;
+    private static final String RANDOM_TRANSFER_BIAS = "randomTransfer.bias";
+
+    private EvmAddressFuzzingFactory() {}
+
+    /**
+     * Initialization operations:
+     * override Ethereum network config (CHAIN_ID_PROP, LAZY_CREATE_PROPERTY_NAME, CONTRACTS_EVM_VERSION_PROP, V_0_34)
+     * initiate key name for signing the Ethereum Transaction (SECP_256K1_SOURCE_KEY)
+     * create account for paying the Ethereum Transaction (RELAYER).
+     * transfer ONE_HUNDRED_HBARS to SECP_256K1_SOURCE_KEY for signing the transaction
+     *
+     * @return array of the initialization operations
+     */
+    public static HapiSpecOperation[] initOperations() {
+        return new HapiSpecOperation[] {
+            overridingThree(
+                    CHAIN_ID_PROP, "298", LAZY_CREATE_PROPERTY_NAME, "true", CONTRACTS_EVM_VERSION_PROP, V_0_34),
+            newKeyNamed(SECP_256K1_SOURCE_KEY).shape(SECP_256K1_SHAPE),
+            cryptoCreate(RELAYER).balance(6 * ONE_MILLION_HBARS).withRecharging(),
+            cryptoTransfer(tinyBarsFromAccountToAlias(GENESIS, SECP_256K1_SOURCE_KEY, ONE_HUNDRED_HBARS))
+                    .via(AUTO_ACCOUNT)
+        };
+    }
+
+    /**
+     * Testing Lazy Create with random EVM addresses.
+     * Operation: creation of random evm addresses and sending hbars through Ethereum Transaction (Lazy Create)
+     *
+     * @param resource config
+     */
+    public static Function<HapiSpec, OpProvider> evmAddressFuzzing(final String resource) {
+        return spec -> {
+            final var props = RegressionProviderFactory.propsFrom(resource);
+
+            final var keys = new RegistrySourcedNameProvider<>(Key.class, spec.registry(), new RandomSelector());
+
+            return new BiasedDelegatingProvider()
+                    .shouldLogNormalFlow(true)
+                    /* --- <inventory> --- */
+                    .withInitialization(onlyEcdsaKeys())
+                    /* ----- CRYPTO ----- */
+                    .withOp(
+                            new TransferToRandomLazyCreate(spec.registry(), keys),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏Perhaps we can make this part of `IdFuzzingProviderFactory` and we'll have this transactions running as part of `AddressAliasIdFuzzing` suite. That would probably make the `EvmAddressFuzzingFactory` and `EthereumTxLazyCreateFuzzing` classes redundant so they can be removed.